src/common/transformations/src/transformations/common_optimizations/leaky_relu_fusion.cpp

```diff
@@ -18,7 +18,7 @@
 ov::pass::LeakyReluFusion::LeakyReluFusion() {
     MATCHER_SCOPE(LeakyReluFusion);
     auto data_pattern = pass::pattern::any_input();
-    auto alpha_pattern = pass::pattern::any_input(pattern::has_static_shape());
+    auto alpha_pattern = pass::pattern::wrap_type<op::v0::Constant>();
```

Comment:
it looks like the previous version worked for subgraphs that can be const folded
If we postpone  fp16 "const folding" of these subgraphs until the CPU phase, does it mean that this transformation won't be executed or CPU runs const folding before the transformation pipeline?
Could you clarify it with @pavel-esir ?

Reply:
Right, this transformation won't be executed. We postpone fp16 Constant->Convert subgraphs constantfolding to first inference.